runtimes/server-interface/workspace.ts

```diff
@@ -19,6 +19,7 @@ export type Workspace = {
     getTextDocument: (uri: string) => Promise<TextDocument | undefined>
     getAllTextDocuments: () => Promise<TextDocument[]>
     getWorkspaceFolder: (uri: string) => WorkspaceFolder | null | undefined
+    getAllWorkspaceFolders: () => WorkspaceFolder[]
```

Comment:
just make it plural. helps discoverability because it will appear next to the singular case. docstrings could be used for both of these cases, meanwhile...


```suggestion
    /** Gets the current(?) workspace folder as decided by ... */
    getWorkspaceFolder: (uri: string) => WorkspaceFolder | null | undefined
    /** Gets all workspace folders known to <something> */
    getWorkspaceFolders: () => WorkspaceFolder[]
```

Reply:
Yes I followed the existing pattern. I will add the doc strings for clarity